pkgs/dart_pad/lib/elements/analysis_results_controller.dart

```diff
@@ -115,16 +109,16 @@ class AnalysisResultsController {
     }
 
     // Add the correction, if any.
-    if (issue.correction.isNotEmpty) {
+    if (issue.correction != null) {
       columnElem.children.add(DivElement()
         ..text = issue.correction
         ..classes.add('message'));
     }
 
-    // TODO: This should likely be named contextMessages.
-    for (final diagnostic in issue.diagnosticMessages) {
-      columnElem.children.add(_createDiagnosticElement(diagnostic, issue));
-    }
+    // // TODO: This should likely be named contextMessages.
+    // for (final diagnostic in issue.diagnosticMessages) {
+    //   columnElem.children.add(_createDiagnosticElement(diagnostic, issue));
+    // }
```

Comment:
Why is this commented out?

Reply:
Same as the other comment - I removed the `contextMessages` field from the info about an analysis issue.